src/gt4py/cartesian/gtc/numpy/npir.py

```diff
@@ -68,11 +68,14 @@ class TemporaryDecl(Decl):
 
     Parameters
     ----------
+    data_dims: Data dimensions sizes (static)
+    dimensions: Flag which cartesian dimensions is present
```

Comment:
```suggestion
    dimensions: Mask which cartesian dimensions are present
```
For me, a flag is for one value and a mask is a collection of flags.